sympy/holonomic/tests/test_holonomic.py

```diff
@@ -91,8 +91,8 @@ def test_addition_initial_condition():
     assert p + q == r
     p = HolonomicFunction(Dx - x + Dx**2, x, 0, [1, 2])
     q = HolonomicFunction(Dx**2 + x, x, 0, [1, 0])
-    r = HolonomicFunction((-4*x**4 - x**3 - 4*x**2 + 1) + (4*x**3 + x**2 + 3*x + 4)*Dx + \
-        (-6*x + 7)*Dx**2 + (4*x**2 - 7*x + 1)*Dx**3 + (4*x**2 + x + 2)*Dx**4, x, 0, [2, 2, -2, 2])
+    r = HolonomicFunction((-x**4 - x**3/4 - x**2 + 1/4) + (x**3 + x**2/4 + 3*x/4 + 1)*Dx + \
+        (-3*x/2 + 7/4)*Dx**2 + (x**2 - 7*x/4 + 1/4)*Dx**3 + (x**2 + x/4 + 1/2)*Dx**4, x, 0, [2, 2, -2, 2])
```

Comment:
It would still be possible to remove the denominators before returning the result. Of course, it is just a cosmetic change, but it would not add much overhead..